### ISSUE TITLE
[Accessibility] Narrator cannot announce the content in RichTextBox.

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#3455]
* .NET Core Version: 
.Net 5.0.100-preview.7.20314.1 from 5.0 master branch

* Have you experienced this same bug with .NET Framework?: 
Yes

**Minimal repro:**

1. Extract “Private_1588_LTR_without_ComboBox.zip” and open in VS, then build and run this application. 
2. Enter "rich" content into RichTextBox, then start Narrator and focus on RichTextBox and observe Narrator behavior.
[Private_1588_LTR_without_ComboBox.zip](https://github.com/dotnet/winforms/files/4810723/Private_1588_LTR_without_ComboBox.zip)

**Expected behavior:**
Narrator should announce the content "rich' in RichTextBox.
![image](https://user-images.githubusercontent.com/45864649/84763618-1316bd00-afff-11ea-9b1f-bdae36ec591f.png)

**Actual behavior:**
Narrator cannot announce the content "rich' in RichTextBox.
![image](https://user-images.githubusercontent.com/45864649/84763561-fc706600-affe-11ea-86be-7a9feeda0926.png)

**Comment by merriemcgaw** [comment]
@Jessie-Zhang01 can you all try out an MFC application with the msedit control on it? @dreddy-work - do you think this could be at all related to which version of the rich text box we're using?

**Comment by Jessie-Zhang01** [comment]
@merriemcgaw in MFC application, Narrator will read its content when the control contains text and switch to that text.
![image](https://user-images.githubusercontent.com/45864649/85939895-a5965500-b94b-11ea-9ec8-4d0a5e69d86a.png)

**Comment by RussKie** [comment]
@Jessie-Zhang01 to confirm, is that a RTB control in the MFC app?

**Comment by Jessie-Zhang01** [comment]
@RussKie we didn't find RTB in the MFC application, we just found Edit Control/Combo Box/Rich Edit 2.0 control and tested them, the Narrator would read their contents.
![image](https://user-images.githubusercontent.com/45864649/85967138-4f312100-b9f4-11ea-8de8-fbf70eccad10.png)

**Comment by merriemcgaw** [comment]
RichEdit2.0 is what we wrap. @dreddy-work  - can you think of anything that we do or don't do with our AccessibleObject here that might be causing the problem? @Jessie-Zhang01 - is this a regression, and if so, when did it show up? Very interested in if we have this issue in 3.1.

Note - this is going to be very high priority for fixing in 5.0.

**Comment by Jessie-Zhang01** [comment]
@merriemcgaw this is not a regression issue, it can reproduce on .Net Core 3.0/.Net Core 3.1/.Net Framework.

In addition, this issue is only reproduced in RichTextBox, DomainUpDown/ListBox/TextBox/MaskedTextBox/NumericUpDown/ComboBox does not reproduce this issue.
![image](https://user-images.githubusercontent.com/45864649/86075633-0c864c00-babb-11ea-9c3c-17eac893a0bc.png)

**Comment by Zheng-Li01** [comment]
Verified the with .NET 6.0.100-preview.4.21181.20, the issue has been fixed that Narrator can announce the content in RichTextBox when setting the RightToLeft property as true, however the Narrator rectangle can't focus on the entire context of RichTextBox contorl, after confirmed with Sergei, filed new GH accessibility issue [4754](https://github.com/dotnet/winforms/issues/4754) to track.
![CoreNewIssue](https://user-images.githubusercontent.com/38325459/113267573-1585a180-9309-11eb-89b7-79813e027b18.gif)

**Comment by John-Qiao** [comment]
Verified in the latest .NET 6.0 P4 build: 6.0.100-preview.4.21224.2, the issue has been fixed that Narrator can announce the content in RichTextBox control as Zheng-Li01 said in previous comment.
![TestResult](https://user-images.githubusercontent.com/45864985/115989413-d50c0180-a5f0-11eb-809e-be49f0e9e51e.png)
